README.md

```diff
@@ -40,7 +40,7 @@ yarn add -D @types/react @types/react-native
 # Getting Started
 
 ```typescript
-import { Calendar } from 'react-native-big-calendar'
+import Calendar from 'react-native-big-calendar'
```

Comment:
Can you restore it? I prefer named export to default export.

Reply:
Sure! 
I've added both exports so it fits every style and updated documentation so cherry-picking is proposed as the first option.

```
import { Calendar } from './Calendar'
export * from './Calendar'
export * from './interfaces'
export default Calendar
```

Reply:
That's great!